tests/__init__.py

```diff
@@ -1,3 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
probably to every file add some copyright like:
```
# 3Di API Client for QGIS, licensed under GPLv2 or (at your option) any later version
# Copyright (C) 2020 by Lutra Consulting for 3Di Water Management
```

Reply:
Done.